ui/src/main/java/com/xmartlabs/bigbang/ui/common/recyclerview/BaseRecyclerViewAdapter.java

```diff
@@ -82,21 +77,42 @@ public void removeItem(@NonNull Object item) {
         });
   }
 
+  /**
+   * Removes a list of items from the data and any registered observers of its removal.
```

Comment:
Are you trying to say: "Removes a list of items from the data and notifies registered observers of its removal."